internal/keys/hasher.go

```diff
@@ -99,14 +108,10 @@ func (t tupleKeysHasher) Append(h hasher) error {
 		key.WriteString("@")
 		key.WriteString(tupleKey.GetUser())
 
-		if err := h.WriteString(key.String()); err != nil {
-			return err
-		}
+		_ = h.WriteString(key.String())
```

Comment:
xxhash package `.WriteString` has two definitions ([safe](https://github.com/cespare/xxhash/blob/main/xxhash_safe.go#L13) and [unsafe](https://github.com/cespare/xxhash/blob/main/xxhash_unsafe.go#L45)) but both always return a `nil` error. We can ignore it

Reply:
Could this maybe be error prone if the underlying lib starts returning non-nil errors in a future release? seems unlikely, but i wonder if it could mask an issue...

Reply:
I suppose it's possible, but it seems unlikely. That'd require xxhash to update its main `write` method, which also always returns nil.

Reply:
removed all of the refactor-related changes. The risk is very small, but why introduce it at all?